### PR TITLE
SAR-3192: New view for already signed up error page

### PR DESCRIPTION
**Pull request by r-melvin**

### CODE REVIEW

File: test/uk/gov/hmrc/vatsignupfrontend/views/principal/AlreadySignedUpSpec.scala [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+  lazy val messagesApi = app.injector.instanceOf[MessagesApi]
+
+  lazy val page = uk.gov.hmrc.vatsignupfrontend.views.html.principal.already_signed_up(
+    )(
+    FakeRequest(),
+    applicationMessages,
+    new AppConfig(configuration, env)
+  )
+
+  "The Principal Cannot use service yet view" should {
```
Comment (palequezal) [inline test/uk/gov/hmrc/vatsignupfrontend/views/principal/AlreadySignedUpSpec.scala]: Need to update the test name to reflect the new view

File: test/uk/gov/hmrc/vatsignupfrontend/views/principal/AlreadySignedUpSpec.scala [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+    val testPage = TestView(
+      name = "Principal Cannot use service yet View",
```
Comment (palequezal) [inline test/uk/gov/hmrc/vatsignupfrontend/views/principal/AlreadySignedUpSpec.scala]: Update name to reflect the new view

File: app/uk/gov/hmrc/vatsignupfrontend/views/principal/already_signed_up.scala.html [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+@import uk.gov.hmrc.vatsignupfrontend.views.html._
+@import uk.gov.hmrc.vatsignupfrontend.views.html.helpers.signOutButton
+
+@()(implicit request: Request[_], messages: Messages, appConfig: AppConfig)
+
+@principal_main_template(title = Messages("principal.already_signed_up.title"), bodyClasses = None, showSignOutLink = false) {
+
+    <h1 class="heading-large">@Messages("principal.already_signed_up.heading")</h1>
+
+    <p>@Messages("principal.already_signed_up.line_1")</p>
```
Comment (JenejVCap) [inline app/uk/gov/hmrc/vatsignupfrontend/views/principal/already_signed_up.scala.html]: you can substitute line_1 with line, when there is only one.

File: test/uk/gov/hmrc/vatsignupfrontend/views/principal/AlreadySignedUpSpec.scala [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+
+class AlreadySignedUpSpec extends ViewSpec {
+
+  val env = Environment.simple()
+  val configuration = Configuration.load(env)
+
+  lazy val messagesApi = app.injector.instanceOf[MessagesApi]
+
+  lazy val page = uk.gov.hmrc.vatsignupfrontend.views.html.principal.already_signed_up(
+    )(
```
[the remark this answers is not in the document]
Comment (r-melvin) [reply test/uk/gov/hmrc/vatsignupfrontend/views/principal/AlreadySignedUpSpec.scala]: It is formatted in line with other specs which call a view with an empty parameter list.

[the remark this answers is not in the document]
Comment (JenejVCap) [reply test/uk/gov/hmrc/vatsignupfrontend/views/principal/AlreadySignedUpSpec.scala]: I see what you're saying, most of them seem to be wrong, i'll raise a ticket to fix them. Go ahead with this.

**Pull request merged**